packages/authenticator/amplify_authenticator/example/lib/resolvers/localized_country_resolver.dart

```diff
@@ -6,7 +6,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
 /// Our custom country string resolver which calls into our localizations class.
-class LocalizedCountryResolver extends CountryResolver {
+
+@Deprecated('Use DialCodeResolver instead')
+typedef CountryResolver = DialCodeResolver;
```

Comment:
This needs to be defined next to the `DialCodeResolver` as well so that's visible to user apps. Placing it here in the example app would not make it usable for end users.